static_src/components/app_container.jsx

```diff
@@ -160,7 +160,7 @@ export default class AppContainer extends React.Component {
               </PageHeader>
             </div>
           </div>
-          <Panel title="Usage and allocation" secondary={ usageSecondary }>
+          <Panel title="App usage and allocation" secondary={ usageSecondary }>
```

Comment:
Could we name this panel `Usage and allocation` and drop the `App`? 
None of the other panels on this page are using the word `App` before the panel title.

Or on the flip side, add `App` to the other panels. On the other pages we are using `Org`, `Space` etc... in the panel titles.